### PR TITLE
ASR-10: Add MongoDb Sink application starter

### PR DESCRIPTION
**Pull request by artembilan**
Fixes spring-cloud-stream-app-starters/app-starters-release#10

### CODE REVIEW

File: mongodb-app-dependencies/pom.xml [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
 		<version>1.2.1.RELEASE</version>
 		<relativePath/>
 	</parent>
+	
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>mongodb-app-dependencies</artifactId>
+	<version>1.1.2.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>mongodb-app-dependencies</name>
+	<description>Spring Cloud Stream Mongodb App Dependencies</description>
```
Comment (garyrussell) [inline mongodb-app-dependencies/pom.xml:18]: What is the purpose of this change? Other poms have this before the parent.

**Review by garyrussell — COMMENTED** [review]
That's all - just a question.

File: mongodb-app-dependencies/pom.xml (hunk 1 above)
Comment (artembilan) [reply mongodb-app-dependencies/pom.xml:18]: IMHO `parent`  is really "parent" and he is the Boss here!

Just try to follow with other conventions, e.g. in Spring Boot: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/pom.xml.
Or Spring Cloud Contract: https://github.com/spring-cloud/spring-cloud-contract/blob/master/spring-cloud-contract-starters/pom.xml

Comment (garyrussell) [reply mongodb-app-dependencies/pom.xml:18]: OK; we just need @sobychacko to sign off on this change and I'll merge.
